package.json

```diff
@@ -10,7 +10,6 @@
     "deploy": "vite build && git push origin --delete gh-pages && git subtree push --prefix dist origin gh-pages"
   },
   "dependencies": {
-    "//@resonai/vera-sdk": "^0.0.11",
     "@resonai/vera-sdk": "file:../resonai2/irocket/client/web/verasdk/"
```

Comment:
don't we need the global? or at least add a comment

Reply:
Reverting this in a later PR